### ISSUE TITLE
Links to each success criterion on the homepage are not very descriptive

### ISSUE THREAD

**Issue by andrewhick** [alphagov/wcag-primer#141]
On the home page, the link text for each criterion is not very descriptive - for example, 'more about 1.3.1' 

This probably fails [WCAG 2.4.9 Link Purpose (Link Only) (AAA)](https://www.w3.org/WAI/WCAG22/Understanding/link-purpose-link-only).
